lib/timber/logger.rb

```diff
@@ -182,8 +182,6 @@ def initialize(*io_devices_and_loggers)
 
       self.level = environment_level
 
-      after_initialize if respond_to?(:after_initialize)
```

Comment:
Why?

Reply:
To remove the DEPRECATION warning that I mentioned above that is polluting the logs.

Reply:
So that's probably the valid solution - `after_initialize if respond_to?(:after_initialize) && ActiveSupport::VERSION::MAJOR < 6`

Taken from https://github.com/Hacker0x01/fluent_logger_rails/issues/4

But as this is our own gem, we don't care about older Rails, I'm going to approve it.